src/components/layers-panel/layers-control-panel.tsx

```diff
@@ -178,12 +208,36 @@ export const LayersControlPanel = ({
     <LayersContainer>
       <LayersList>{renderLayers(layers)}</LayersList>
       <InsertButtons>
-        <PlusButton buttonSize={ButtonSize.Small} onClick={onLayerInsertClick}>
+        <ActionIconButton Icon={PlusIcon} size={ButtonSize.Small} onClick={onLayerInsertClick}>
           Insert layer
-        </PlusButton>
-        <PlusButton buttonSize={ButtonSize.Small} onClick={onSceneInsertClick}>
+        </ActionIconButton>
+        <ActionIconButton Icon={PlusIcon} size={ButtonSize.Small} onClick={onSceneInsertClick}>
           Insert scene
-        </PlusButton>
+        </ActionIconButton>
+
+        <PanelHorizontalLine />
+
+        {showLogin && (
+          <ActionIconButtonContainer>
+            <ActionIconButton Icon={ImportIcon} style={'disabled'} size={ButtonSize.Small} onClick={onArcGisLoginClick}>
+              Login to ArcGIS
+            </ActionIconButton>
+            <EsriStyledImage />
+          </ActionIconButtonContainer>
+        )}
+        {showImport && (
+          <ActionIconButtonContainer>
+            <ActionIconButton Icon={ImportIcon} style={'active'} size={ButtonSize.Small} onClick={onArcGisImportClick}>
+              Import from ArcGIS
+            </ActionIconButton>
+            <EsriStyledImage />
+          </ActionIconButtonContainer>
+        )}
+        {showLogout && (
```

Comment:
Extra complexity is not needed
```suggestion
        {isLoggedIn&& (
```